calico.go

```diff
@@ -125,12 +125,19 @@ func cmdAdd(args *skel.CmdArgs) error {
 				return err
 			}
 
+			// Get labels
+			labels := make(map[string]string)
```

Comment:
I think my preference here would be not to use make unless declaring an explicit capacity up front (which we're not doing here).

So, I'd initialize like so:

```go
labels := map[string]string{}
```